app/views/operations/index.html.erb

```diff
@@ -0,0 +1,13 @@
+
+<h1>Operations</h1>
+
```

Comment:
- [ ] As I mentioned above, This page (`operations#index` page) is supposed to be your transaction page. 
As per these [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=a%20new%20category.-,Transactions%20page,-For%20a%20given), Please edit this page to serve the following:
  - For a given category, the list of transactions is presented, ordered by the most recent.
  - At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
  - There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.